### PR TITLE
[WIP] Python Cilent MVP

### PR DESCRIPTION
**Pull request by darioalessandro**

### CODE REVIEW

File: .github/CI.yml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+permissions:
+  contents: read
+
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        target: [x86_64, x86, aarch64, armv7, s390x, ppc64le]
```
Comment (hubcio) [inline .github/CI.yml]: i believe you can skip s390x in the pipeline, it was discontinued in 2004 😄 same goes to ppc64le, I suggest leaving only:

x86_64, i386/i686, apple-aarch64-darwin, armv7, aarch64/arm64v8

Comment (hubcio) [reply .github/CI.yml]: remark: I didn't notice that it was generated :D later on, stick to same targets as our CI in the main rust repo

Comment (darioalessandro) [reply .github/CI.yml]: sgtm @hubcio 😄

File: src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+use pyo3::prelude::*;
```
Comment (darioalessandro) [inline src/lib.rs]: the actual iggy wrappers go here, this is the default generated project

File: Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+[package]
+name = "iggy-py"
```
Comment (darioalessandro) [inline Cargo.toml:2]: @spetz @hubcio how should we call this library? Iggy is taken :( https://pypi.org/project/iggy/

**Pull request merged**